src/containers/EditEventPage.js

```diff
@@ -28,6 +30,17 @@ const mapStateToProps = (state, props) => ({
   event: state.events.find(event => event.id === props.match.params.id),
 });
 
+EditEventPage.propTypes = {
+  editEvent: PropTypes.func.isRequired,
+  event: PropTypes.shape({
```

Comment:
Is this the same event as in `EventForm.js` file? If so, then please, create propType config for that similar as I suggested for the history object and the reuse that.